roster2/bundle/src/bundle/roster.properties

```diff
@@ -94,6 +94,7 @@ title_msg_permissions = Define permissions for the roles in the current site.
 no_participants = No results found
 
 profile_link_text = Click Here
```

Comment:
"Click here" is the link text? What can it be instead?  "More"? Or the person's name?

Reply:
Could be I suppose. The name could get long though